app/src/components/ui/Radius.vue

```diff
@@ -24,31 +24,48 @@
             },
         },
         data(): {
-            selectedRadius: string,
+            newSelectedRadius: string,
+            radii: Array<{text: string, value: string}>
         } {
             return {
-                selectedRadius: this.$store.state.radiusSearchValue || '',
+                newSelectedRadius: '',
+                radii: [
+                    {
+                        text: 'Überall',
+                        value: '',
+                    },
+                    {
+                        text: '5 km',
+                        value: '5km',
+                    },
+                    {
+                        text: '10 km',
+                        value: '10km',
+                    },
+                    {
+                        text: '25 km',
+                        value: '25km',
+                    },
+                    {
+                        text: '50 km',
+                        value: '50km',
+                    },
+                ]
             };
         },
         watch: {
-            selectedRadius(newValue, oldValue): void {
-                this.setRadiusSearchValue(newValue);
+            newSelectedRadius(newValue, oldValue): void {
+                this.setSelectedRadius(newValue);
                 if (this.$route.name === 'resultPage') {
-                    this.$router.push({
-                    name: 'resultPage',
-                    query: {
-                        ...this.$route.query,
-                        radius: newValue,
-                    }
-                    });
+                   this.$store.dispatch('updateURIFromState');
                 }
             },
         },
-        methods: {
-            ...mapActions(['setRadiusSearchValue']),
-        },
         computed: {
-            ...mapState(['radii'])
+            ...mapState(['selectedRadius'])
+        },
+        methods: {
+            ...mapActions(['setSelectedRadius']),
```

Comment:
If I understand this correctly, the actions are injected at runtime, and their definition is in the actions file in the respective component dir - the mapping is handled by vuex and uses the identifier given in this array ? 

Do you think it might be beneficial to import the identifiers from a registry object or enum or something like that to prevent typos as a potential error source? This could additionally emphasize the fact, that the functionality is implemented in `actions.ts`.

Reply:
Correct. We event generate the mapState and mapActions functions at runtime to use the correct namespace

https://github.com/hochschule-darmstadt/einander-helfen/pull/62/files/40b402e0cea62c8f08485318748b5101b6b675d7#diff-9262a6da29f5774642cb72534f2956a9R14-R15

I would gladly accept a PR for this but don´t think the has priority.